content/sensu-go/6.1/observability-pipeline/observe-schedule/subscriptions.md

```diff
@@ -0,0 +1,158 @@
+
+*<p style="text-align:center">Sensu check execution based on subscriptions</p>*
+
+Sensu subscriptions are equivalent to topics in a traditional publish/subscribe system.
+Sensu entities become subscribers to these topics via the strings you specify with the agent `subscriptions` flag.
+Sensu checks have a `subscriptions` attribute, where you specify strings to indicate which subscribers will execute the checks.
+For Sensu to execute a check, the check definition must include a subscription that matches the subscription of at least one Sensu entity.
+
+As loosely coupled references, subscriptions avoid the fragility of traditional host-based monitoring systems.
+Subscriptions allow you to configure check requests in a one-to-many model for entire groups or subgroups of entities rather than a traditional one-to-one mapping of configured hosts or observability checks.
+
+## Configure subscriptions
+
+For Sensu to execute a check, the check definition must include a subscription that matches a subscription for at least one Sensu entity.
+In other words, subscriptions are configured for both checks and agents:
+
+- To configure subscriptions for a check, add one or more subscription names in the [check `subscriptions` attribute][15].
+- To configure subscriptions for an agent, configure the [`subscriptions`][2] by specifying the subscriptions that include the checks the agent's entities should execute.
+
+The Sensu backend [schedules][13] checks once per interval for each agent with a matching subscription.
+For example, if you have three agents configured with the `system` subscription, a check configured with the `system` subscription results in three monitoring events per interval: one check execution per agent per interval.
+
+In addition to the subscriptions defined in the agent configuration, Sensu agent entities subscribe automatically to subscriptions that match their [entity `name`][10].
+For example, an agent entity with `name: "i-424242"` subscribes to check requests with the subscription `entity:i-424242`.
+This makes it possible to generate ad hoc check requests that target specific entities via the API.
+
+{{% notice note %}}
+**NOTE**: You can directly add, update, and delete subscriptions for individual entities via the backend with [sensuctl](../../../sensuctl/create-manage-resources/#update-resources), the [entities API](../../../api/entities/), and the [web UI](../../../web-ui/view-manage-resources/#manage-entities).
+{{% /notice %}}
+
+## Publish checks
+
+If you want Sensu to automatically schedule and execute a check according to its subscriptions, set the [`publish` attribute][12] to `true` in the check definition.
+
+You can also manually schedule [ad hoc check execution][11] with the [check API][16], whether the `publish` attribute is set to `true` or `false`.
+To target the subscriptions defined in the check, include only the check name in the request body (e.g. `"check": "check-cpu"`).
```

Comment:
```suggestion
To target the subscriptions defined in the check, include only the check name in the request body (e.g. `"check": "check_cpu"`).
```

Suggest renaming `check-cpu` to `check_cpu` for consistency with check name used in the above diagram.
Perhaps consider renaming checks in the diagram to use consistent `snake_case` or `camelCase` names?